src/main/java/com/iotiq/user/internal/UserService.java

```diff
@@ -53,8 +54,20 @@ public boolean existByUserName(String username) {
         return userRepository.existsByAccountInfoUsername(username);
     }
 
+    @Transactional(readOnly = true)
+    public boolean existByEmail(String email) {
+        return userRepository.existsByPersonalInfoEmail(email);
+    }
+
+    public int countUsersWithEmailAndNotId(UUID id, String email) {
```

Comment:
This method calls a method with the same signature therefore seems unnecessary to me.

Reply:
Noted!